docs/platform_introduction/core_elements/services.md

```diff
@@ -1,12 +1,12 @@
 # Services
 
-Every study in *osparc* is composed of at least one so-called service, though most studies will contain multiple services.
+Every study in *osparc* is composed of at least one so-called service (also called node), though most studies will contain multiple services.
```

Comment:
MINOR: what about ``(represented as nodes in a graph)`` ?

Reply:
I refer to them as nodes in other places (actually it originated because you pointed out that we have too many things called "services" =)